src/main/java/uk/gov/pay/webhooks/app/IdleConnectionMonitorThread.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.pay.webhooks.app;
+
+import org.apache.http.conn.HttpClientConnectionManager;
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+
+import java.util.concurrent.TimeUnit;
+
+public class IdleConnectionMonitorThread extends Thread {
+    private final HttpClientConnectionManager connMgr;
+    private volatile boolean shutdown;
+    private final long idleConnectionTimeToLive;
```

Comment:
This could be a `Duration` (rather than a `long`) and just be converted every time it’s run.